packages/@react-spectrum/numberfield/stories/NumberField.stories.tsx

```diff
@@ -228,6 +228,13 @@ storiesOf('NumberField', module)
   .add(
     'focus events',
     () => render({onBlur: action('onBlur'), onFocus: action('onFocus'), onFocusChange: action('onFocusChange'), onKeyDown: action('onKeyDown'), onKeyUp: action('onKeyUp')})
+  )
+  .add(
+    'input dom events',
+    () => render({
+      onCopy: action('onCopy'), onCut: action('onCut'), onPaste: action('onPaste'), onCompositionStart: action('onCompositionStart'), onCompositionEnd: action('onCompositionEnd'),
+      onCompositionUpdate: action('onCompositionUpdate'), onSelect: action('onSelect'), onBeforeInput: action('onBeforeInput'), onInput: action('onInput')
```

Comment:
Noticed some oddities with how the events are getting fired here: 
- `onSelect` seems to fire whenever the input is changed, not just when text is selected. This happens in our TextField as well.
- `onBeforeInput` doesn't seem to ever fire but it works just fine in our textfield 
The other events seems to fire just fine

Reply:
Possible alternative:
If we are unhappy with those two event being broken for NumberField, we can override them in useNumberField for now until we figure out why they don't work

Reply:
Thank you so much for looking at this @LFDanLu, much appreciated!!  Great catch, I'm happy to make that (or any other) update you is the best path, just let me know what you prefer 👍 Thanks again!

Reply:
Sure thing, no problem, thank you for the PR. If you'd like to take a shot at debugging those broken events definitely feel free, but don't feel pressured to do so since it is a bit tangential to the original issue which you've fixed. The team can handle the debugging/followup if this goes in as is.

I'll ask the team tomorrow what they feel about these broken events and get a consensus for a path forward